app.js

```diff
@@ -0,0 +1,74 @@
+var arr = 
+[
+  {
+    name: "Lion King",
+    license: [2011, 2013]
+  },
+
+  {
+    name: "Iron Man",
+    license: [2005,2010]
+  },
+  {
+    name: "Life of Pi",
+    license: [2008,2014]
+  },
+  {
+    name: "Another Movie",
+    license: [2011,2013]
+  }
+];
+
+
+var frequency = {};
+
+function main(arr)
+{
+	countYearFrequency(arr)
+	maxYears = getMaxYear(frequency)
+	return maxYears
+}
+
+function countYearFrequency(arr) 
+{
+	arr.forEach(updateYearFrequency);     
+}
+
+function updateYearFrequency(item)
+{
+	startOfRange = item.license[0]
+	endOfRange = item.license[1]
+	
+	var i;
+	for(i = startOfRange; i<=endOfRange; i++ )
```

Comment:
`for(let i= ...)` 
No need to declare `i` above the loop

Read up on scope of `var`

and differences between `var`, `const`, `let`